actions/matrixsynapse

```diff
@@ -35,6 +38,10 @@ def parse_arguments():
     subparsers.add_parser('post-install', help='Perform post install steps')
     subparsers.add_parser('enable', help='Enable matrix-synapse service')
     subparsers.add_parser('disable', help='Disable matrix-synapse service')
+    subparsers.add_parser('enable-user-registrations', help='Allow users to' \
+                                                            'sign up to this matrixsynapse* pod without an invitation.')
+    subparsers.add_parser('disable-user-registrations', help='Allow only, ' \
```

Comment:
Again, rather minor, but imo nicer to have enable/disable as subcommands of a single command that handles registrations. See solution in current code.